### PR TITLE
Ticket sending

### PR DESCRIPTION
**Pull request by KwikKill**
This PR :
- Deref ticket in tournament/me in order to get the ticket id
- Change ticket `/get/{username}/{token}` to `/get/{id}/{token}`
- Add a generate ticket pdf from ticket method
- Add a send ticket by mail method
- Send ticket on payment

The ticket generation use reportlab. This is pretty hard to maintains imo but no serious alternatives were available. I tried a bunch of tools to convert html to pdf so we could had only a html template but they were all not supporting CSS3 or based on webdrivers hard to maintains.

### CODE REVIEW

File: insalan/tickets/tests.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def create_ticket(
     token: str,
     tourney: Tournament,
     ticket_status: Ticket.Status = Ticket.Status.VALID,
-) -> None:
+) -> int:
     """
     Create a ticket for the given username and token.
```
Comment (Lugrim) [inline insalan/tickets/tests.py:35]: Could we document the return value?

**Review by Lugrim — COMMENTED** [review]
Only the sanitation is actually mandatory, my other suggestions mostly are for code quality

**Review by Lugrim — APPROVED** [review]
AFAIK my concerns have been resolved, good job!

File: insalan/user/models.py [hunk 2]
```diff
@@ -209,4 +209,29 @@ def send_kick_mail(user_object: User, team_name: str):
             fail_silently=False,
         )
 
+    @staticmethod
+    def send_ticket_mail(user_object: User, ticket: str):
+        """
+        Send a mail to a user that has been kicked.
```
Comment (Lymkwi) [inline insalan/user/models.py:215]: Wrong description here btw, and it's also in #128

**Review by Lymkwi — APPROVED** [review]
Just one nitpick, but it could be fixed in #128 .

**Pull request merged**